bin/create_venv.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+THIS_DIR=$(dirname $(readlink -f $0))
+PROJ_ROOT=${THIS_DIR}/..
+
+# Set different virtual environment paths so that these don't clash when
+# mounting the code in a development container
+VENV_PATH="undetected"
+if [[ -z "$FAABRIC_DOCKER" ]]; then
+    VENV_PATH="${PROJ_ROOT}/venv-bm"
+else
+    VENV_PATH="/code/faabric/venv"
+fi
+
+PIP=${VENV_PATH}/bin/pip3
+
+function pip_cmd {
+    source ${VENV_PATH}/bin/activate && ${PIP} "$@"
+}
+
+pushd ${PROJ_ROOT} >> /dev/null
+
+if [ ! -d ${VENV_PATH} ]; then
+    python3 -m venv ${VENV_PATH}
+fi
+
+pip_cmd install -U pip
+pip_cmd install -r requirements.txt
```

Comment:
`wheel` should be installed before requirements.txt, otherwise installing packages via requirements falls back to a much slower path

Reply:
Yeah, thanks for pointing this out! I will request a review from you now, see if you spot something else.